pages/issues/[id].js

```diff
@@ -14,6 +15,19 @@ const Issue = () => {
     `/api/issues/${id}`
   );
 
+  const [favorites, setFavorites] = useLocalStorage("favoriteContents", []);
```

Comment:
What happened to the hook?

Reply:
Nevermind it's on the other PR. Might get some merge conflicts on this so watch out

Reply:
Hi @kerimhudson,
It doesn't complain of a conflict, does it mean it's fine? Or it could still mess the code up?